notebooks/file_calibration.ipynb

```diff
@@ -61,6 +61,7 @@
     "from hera_notebook_templates.data import DATA_PATH as HNBT_DATA\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        cross_antpos = {
</pre><p>Ideally, we'd calibrate en for a given baseline even if we don't have ne.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_notebook_templates/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Reply:
Generalized this to calibrate with one cross-pol even if the other is flagged